rush.json

```diff
@@ -648,6 +648,11 @@
       "projectFolder": "sdk/servicebus/service-bus",
       "versionPolicyName": "client"
     },
+    {
+      "packageName": "@azure/storage-common",
+      "projectFolder": "sdk/storage/storage-common",
+      "versionPolicyName": "client"
+    },
```

Comment:
Why added this package? Currently the storage-common is only a place to put common used codes instead of a package. 

To add a commone package, we'd need to change release pipeline at the same time.